mne/evoked.py

```diff
@@ -611,6 +614,13 @@ def get_peak(self, ch_type=None, tmin=None, tmax=None, mode='abs',
                                'must not be `None`, pass a sensor type '
                                'value instead')
 
+        if merge_grads:
+            if ch_type != 'grad':
+                raise ValueError('Channel type must be grad for merge_grads')
+            elif mode == 'neg':
+                raise ValueError('Negative mode does not make sense with '
```

Comment:
slightly more explicit to say `Negative mode (mode='neg')` does not ...` because it tells the user the other thing they could change